src/server/cmd/pachd/main.go

```diff
@@ -168,6 +167,7 @@ func doEnterpriseMode(config interface{}) (retErr error) {
 				return err
 			}
 			authclient.RegisterAPIServer(externalServer.Server, authAPIServer)
+			env.SetAuthServer(authAPIServer)
```

Comment:
I find it slightly tidier to give these servers to the serviceenv as we create them, instead of having one big initialize call at the end, but ymmv.